File: lc_running-sum-of-1d-array.py
# Question: https://leetcode.com/problems/running-sum-of-1d-array/

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        ans = []
        ans.append(nums[0])
        for i in range(1,len(nums)):
            ans.append(nums[i] + ans[i-1])
        return ans

File: lc_running-sum-of-1d-array.js
// Question: https://leetcode.com/problems/running-sum-of-1d-array/

/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function (nums) {
  const ans = [];
  ans.push(nums[0]);
  for (let i = 1; i < nums.length; i++) {
    ans.push(nums[i] + ans[i - 1]);
  }
  return ans;
};
